surfnperf.js

```diff
@@ -356,6 +356,57 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (browser-reported time for the first pixels to be rendered on the page)
+   *
+   * @returns {integer} time in ms
+   * @memberOf SurfNPerf
+   */
+  SNPProto.getFirstPaint = function() {
+    if(this.chromeLoadTimes()) {
+      return this.chromeLoadTimes().firstPaintTime * 1000 - this.performanceTiming().navigationStart;
```

Comment:
instead of doing subtraction, use the `duration()` function and pass in the `options` argument from `getFirstPaint` as the 3rd argument so users can pass `{decimalPlaces:3}` if they want